src/test/regress/spec/isolation_blocking_move_single_shard_commands.spec

```diff
@@ -31,7 +32,8 @@ step "s1-begin"
 
 step "s1-move-placement"
 {
-    	SELECT master_move_shard_placement((SELECT * FROM selected_shard), 'localhost', 57637, 'localhost', 57638, shard_transfer_mode:='block_writes');
+    SET citus.defer_drop_after_shard_move to off;
```

Comment:
Why is this needed? I'd expect it not to be necessary with the shardstate != 4 changes.